examples/gatsby/src/templates/page.tsx

```diff
@@ -1,7 +1,11 @@
 import * as React from 'react';
 import { graphql } from 'gatsby';
 import { BuilderComponent } from '@builder.io/react';
-import '../components/CodeBlock';
+/**
+ * Import all your custom components here If you're using any with builder.io,
```

Comment:
would be good to have an example here instead as most people won't understand this

e.g. add ../components/custom-block-example.ts

and in that example link to the docs for custom react components and the design system example